src/utils/counter.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+const counter = (count) => {
```

Comment:
This function looks a little bit odd. There should be a way to make this shorter and more effective. For example, this function will stop counting at 10,000.

Take a look at this snippet I created. It should work for what you are trying to do.

https://repl.it/@bamideleboluwatife/SizzlingRunnyReality

Reply:
You can optimize it if you want to.